src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue6784.cs

```diff
@@ -13,42 +15,38 @@ public Issue6784(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "ShellItem.CurrentItem is not set when selecting shell section aggregated in more tab";
 
-	//[Test]
-	//public void CurrentItemIsSetWhenSelectingShellSectionAggregatedInMoreTab()
-	//{
-	//	App.WaitForElement(x => x.Class("UITabBarButton").Marked("More"));
-	//	App.Tap(x => x.Class("UITabBarButton").Marked("More"));
-
-	//	App.WaitForElement(x => x.Class("UITableViewCell").Text("Tab 5"));
-	//	App.Tap(x => x.Class("UITableViewCell").Text("Tab 5"));
-
-	//	App.WaitForElement(x => x.Text("Success"));
-	//}
-
-	//[Test]
-	//public void MoreControllerOpensOnFirstClick()
-	//{
-	//	App.WaitForElement(x => x.Class("UITabBarButton").Marked("More"));
-	//	App.Tap(x => x.Class("UITabBarButton").Marked("More"));
-
-	//	App.WaitForElement(x => x.Class("UITableViewCell").Text("Tab 5"));
-	//	App.Tap(x => x.Class("UITableViewCell").Text("Tab 5"));
-
-	//	App.Tap(x => x.Class("UITabBarButton").Marked("Tab 4"));
-	//	App.WaitForElement("Tab 4 Content");
-
-	//	App.Tap(x => x.Class("UITabBarButton").Marked("More"));
-	//	App.WaitForElement(x => x.Class("UITableViewCell").Text("Tab 6"));
-	//}
-
-	//[Test]
-	//public void MoreControllerDoesNotShowEditButton()
-	//{
-	//	App.WaitForElement(x => x.Class("UITabBarButton").Marked("More"));
-	//	App.Tap(x => x.Class("UITabBarButton").Marked("More"));
+	[Test]
+	public void CurrentItemIsSetWhenSelectingShellSectionAggregatedInMoreTab()
+	{
+		App.WaitForElement("More");
+		App.Tap("More");
+		App.WaitForElement("Tab 11");
+		App.Tap("Tab 11");
+		App.WaitForElementTillPageNavigationSettled("Success");
+	}
 
-	//	App.WaitForElement(x => x.Class("UITableViewCell").Text("Tab 5"));
+	[Test]
+	public void OneMoreControllerOpensOnFirstClick()
+	{
+		App.WaitForElement("More");
+		App.Tap("More");
+		App.WaitForElement("Tab 11");
+		App.Tap("Tab 11");
+		App.WaitForElement("Tab 4");
+		App.Tap("Tab 4");
+		App.WaitForElement("More");
+		App.Tap("More");
+		App.WaitForElement("Tab 12");
+		App.Tap("Tab 12");
+	}
 
-	//	Assert.AreEqual(App.Query(x => x.Marked("Edit")).Count(), 0);
-	//}
-}
\ No newline at end of file
+	[Test]
+	public void TwoMoreControllerDoesNotShowEditButton()
```

Comment:
Failing on macOS:
<img width="1160" alt="image" src="https://github.com/user-attachments/assets/bc18cbc0-8edd-4711-8a55-fbe89438a9b7">

Reply:
@jsuarezruiz This issue arises because Appium was able to locate the edit element from the app menu items in the top menu bar on macOS. I have added a condition to address this problem. Please review and share your feedback.